rasterio/_example.pyx

```diff
@@ -1,10 +1,10 @@
 # cython: boundscheck=False
 
-import numpy
-cimport numpy
+import numpy as np
+cimport numpy as np
```

Comment:
Looks like those imports aren't necessary since we're using cython's typed memory views.